### DISCUSSION TITLE
Is it possible to set the Blazorise Bar to collapsed by default?

### DISCUSSION THREAD

**Discussion by luis0795** [Megabit/Blazorise#4051] [Q&A] [answered]
Afternoon all!

Just wondering if anyone knows how I can set my sidebar collapsed by default, so the user will expand only when need to check the info inside!

I'm refering to this component:
https://blazorise.com/docs/components/bar/

thanks in advance!

**Comment by David-Moreira** [comment, marked as answer]
Hello,

If you scroll down on the page you've sent, you see a few examples of "Togglers". To start collapsed I guess using the `Visible` parameter should work, have you tried it?

**Reply by luis0795** [reply]
Thank U David, missed that works perfectly, don't know how I missed that!

![image](https://user-images.githubusercontent.com/35500932/186182283-b5c5aff5-ae4f-42d6-8d41-0de7992c72df.png)
